Index.html

```diff
@@ -0,0 +1,231 @@
+    <link rel="stylesheet" href="styles.css">
+    <title>Twitter</title>
+  </head>
+  <body>
+   <header>
+      <nav>
+          <ul>
+              <li>
+                  <i class="twitter"></i>
+              </li>
+              <li> 
+              <div class="Logo" ></div> 
+              <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewbox="10 0 85 30" fill="rgb(29, 155, 240)"> 
+ 98 13.255-13.254 0-.2-.005-.402-.014-.602.91-.658 1.7-1.477                <g>
+                    <path d="M23.643 4.937c-.835.37-1.732.62-2.675.733.962-.576 1.7-1.49 2.048-2.578-.9.534-1.897.922-2.958 1.13-.85-.904-2.06-1.47-3.4-1.47-2.572 0-4.658 2.086-4.658 4.66 0 .364.042.718.12 1.06-3.873-.195-7.304-2.05-9.602-4.868-.4.69-.63 1.49-.63 2.342 0 1.616.823 3.043 2.072 3.878-.764-.025-1.482-.234-2.11-.583v.06c0 2.257 1.605 4.14 3.737 4.568-.392.106-.803.162-1.227.162-.3 0-.593-.028-.877-.082.593 1.85 2.313 3.198 4.352 3.234-1.595 1.25-3.604 1.995-5.786 1.995-.376 0-.747-.022-1.112-.065 2.062 1.323 4.51 2.093 7.14 2.093 8.57 0 13.255-7.02.323-2.41z"></path>
+                </g>
+              </svg>
+              </li>
+              <li>
+                  <div class="header-icon"><i class="home"></i></div>
+                  <a href="#home"><strong>Home</strong></a>
+                  <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewbox="-5 -10 50 50" fill="rgb(29, 155, 240)">
+                    <g>
+                        <path
+                            d="M22.58 7.35L12.475 1.897c-.297-.16-.654-.16-.95 0L1.425 7.35c-.486.264-.667.87-.405 1.356.18.335.525.525.88.525.16 0 .324-.038.475-.12l.734-.396 1.59 11.25c.216 1.214 1.31 2.062 2.66 2.062h9.282c1.35 0 2.444-.848 2.662-2.088l1.588-11.225.737.398c.485.263 1.092.082 1.354-.404.263-.486.08-1.093-.404-1.355zM12 15.435c-1.795 0-3.25-1.455-3.25-3.25s1.455-3.25 3.25-3.25 3.25 1.455 3.25 3.25-1.455 3.25-3.25 3.25z">
+                        </path>
+                    </g>
+                </svg>
+              </li>
+              <li>
+                  <div class="header-icon"><i class="hashtag"></i></div>
+                  <a href="">Explore</a>
+                  <svg viewbox="-5 -10 50 50" aria-hidden="true" class="r-18jsvk2 r-4qtqp9 r-yyyyoo r-lwhw9o r-dnmrzs r-bnwqim r-1plcrui r-lrvibr"><g><path d="M21.53 20.47l-3.66-3.66C19.195 15.24 20 13.214 20 11c0-4.97-4.03-9-9-9s-9 4.03-9 9 4.03 9 9 9c2.215 0 4.24-.804 5.808-2.13l3.66 3.66c.147.146.34.22.53.22s.385-.073.53-.22c.295-.293.295-.767.002-1.06zM3.5 11c0-4.135 3.365-7.5 7.5-7.5s7.5 3.365 7.5 7.5-3.365 7.5-7.5 7.5-7.5-3.365-7.5-7.5z"></path></g></svg>
+              </li>
+              <li>
+                  <div class="header-icon"><i class="bell"></i></div>
+                  <a href="">Notifications</a>
+                  <svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewbox="-5 -10 50 50"  fill="rgb(29, 155, 240)">
```

Comment:
svg's should be in their own file altogether and not in the html, then in the html have an <img> and then point the src to the svg